tests/test_cargo_install.rs

```diff
@@ -543,3 +543,24 @@ test!(installs_from_cwd_by_default {
     assert_that(cargo_process("install"), execs().with_status(0));
     assert_that(cargo_home(), has_installed_exe("foo"));
 });
+
+test!(do_not_rebuilds_on_local_install {
+    let p = project("foo")
+        .file("Cargo.toml", r#"
+            [package]
+            name = "foo"
+            version = "0.1.0"
+            authors = []
+        "#)
+        .file("src/main.rs", "fn main() {}");
+    assert_that(p.cargo_process("build").arg("--release"),
+                execs().with_status(0));
+
+    assert_that(cargo_process("install").arg("--path").arg(p.root()),
```

Comment:
Ah yeah sorry I just meant that `p.cargo(...)` was more idiomatic, but for `install` it's probably fine to just use this (as it's what the other install tests do)